src/ophys_etl/modules/roi_cell_classifier/compute_classifier_artifacts.py

```diff
@@ -0,0 +1,275 @@
+import pathlib
+import os
+import time
+from typing import Dict
+
+import h5py
+import json
+import numpy as np
+import PIL
+
+from argschema import ArgSchema, ArgSchemaParser, fields
+from ophys_etl.modules.segmentation.graph_utils.conversion import \
+    graph_to_img
+from ophys_etl.types import ExtractROI
+from ophys_etl.utils.array_utils import normalize_array
+from ophys_etl.utils.rois import (
+    sanitize_extract_roi_list,
+    extract_roi_to_ophys_roi)
+
+
+class ClassifierArtifactsInputSchema(ArgSchema):
+    # Input data locations.
+    video_path = fields.InputFile(
+        required=True,
+        description="Path to motion corrected(+denoised) video.",
+    )
+    roi_path = fields.InputFile(
+        required=True,
+        description="Path to json file containing detected ROIs",
+    )
+    graph_path = fields.InputFile(
+        required=True,
+        description="Path to pickle file containing full movie graph.",
+    )
+
+    # Output Artifact location.
+    out_dir = fields.OutputDir(
+        required=True,
+        description="Output directory to put artifacts.",
+    )
+
+    # Artifact generation settings.
+    low_quantile = fields.Float(
+        required=False,
+        defalut=0.2,
+        description="Low quartile to saturate/clip to.",
+    )
+    high_quantile = fields.Float(
+        required=False,
+        defalut=0.99,
+        description="High quartile to saturate/clip to.",
+    )
+    cutout_size = fields.Int(
+        required=False,
+        defalut=128,
+        description="Size of square cutout.",
+    )
+
+
+class ClassifierArtifactsGenerator(ArgSchemaParser):
+    """Create cutouts from the average, max, correlation projects for each
+    detected ROI. Additionally return the ROI mask. Store outputs as 8bit
+    pngs.
+    """
+
+    default_schema = ClassifierArtifactsInputSchema
+
+    def run(self):
+        self.logger.name = type(self).__name__
+        self.logger.setLevel(self.args["log_level"])
+
+        ophys_etl_commit_sha = os.environ.get("OPHYS_ETL_COMMIT_SHA",
+                                              "local build")
+        self.logger.info(f"OPHYS_ETL_COMMIT_SHA: {ophys_etl_commit_sha}")
+        t0 = time.time()
+
+        out_dir = pathlib.Path(self.args["out_dir"])
+        if not out_dir.exists():
+            out_dir.mkdir()
```

Comment:
Have you tested this? `argschema` purports to create an `OutputDir` if it doesn't exist, so I think this code block is not needed.